### ISSUE TITLE
No-nav engage pages

### ISSUE THREAD

**Issue by caldav** [canonical-web-and-design/test-and-learn#5]
### Test info

| | |
| --- | --- |
|**User problem to solve**  | Users might be overwhelmed by the amount of choices offered in the nav |
|**Business problem to solve**  | A lot of users who land on the engage pages do not convert |
|**Objectives**  | Increase the number of leads from engage pages |
|**Hypothesis**  | Removing the navigation from engage pages to increase conversion rate. |
|**Audience**  | All visits to engage pages ubuntu.com/engage/* |
|**Stakeholders**  | David C |

### Test duration

[A/B test duration calculator](https://vwo.com/ab-split-test-duration/)

| | |
| --- | --- |
|**Traffic**  | TBC |
|**Current conversion rate**  | TBC |
|**Improvement**  | TBC |
|**Days**  | TBC |

### Test results

| | |
| --- | --- |
|**Data**  | [Google Optimize Link - Test 01 with webinars & whitepapers (ended)](https://optimize.google.com/optimize/home/#/accounts/1601306181/containers/7212199/experiments/139/report)<br/>[Google Optimize Link - Test 02 with whitepapers](https://optimize.google.com/optimize/home/#/accounts/1601306181/containers/7212199/experiments/154/report) |

|**Learnings**  | Insights from the test |
|**Next steps**  | Follow up tests, hard code winner, etc |

**Variations**

Mobile Control
<img width="357" alt="Screenshot 2019-08-20 at 14 52 15" src="https://user-images.githubusercontent.com/47972694/63353170-41cdf380-c35a-11e9-8990-ac788175dc9b.png">

Mobile Variation
<img width="358" alt="Screenshot 2019-08-20 at 14 52 05" src="https://user-images.githubusercontent.com/47972694/63353168-41355d00-c35a-11e9-8830-08f65a5c898d.png">

Desktop Control
<img width="1124" alt="Screenshot 2019-08-20 at 14 52 56" src="https://user-images.githubusercontent.com/47972694/63353173-41cdf380-c35a-11e9-98a8-d8a999df069a.png">

Desktop Variation
<img width="1119" alt="Screenshot 2019-08-20 at 14 52 25" src="https://user-images.githubusercontent.com/47972694/63353172-41cdf380-c35a-11e9-9633-a57ea236d93d.png">

**Comment by caldav** [comment]
Examples:
![Screenshot_2019-07-11 A guide to edge computing and the tools you need Ubuntu](https://user-images.githubusercontent.com/18480003/61053437-d35a5500-a3ed-11e9-8e81-32bbdadbca66.png)
![Screenshot_2019-07-11 How to integrate Ubuntu Desktop with Active Directory Ubuntu(1)](https://user-images.githubusercontent.com/18480003/61053440-d3f2eb80-a3ed-11e9-8272-52d4398e9b8b.png)
![Screenshot_2019-07-11 The future of mobile connectivity Ubuntu](https://user-images.githubusercontent.com/18480003/61053441-d3f2eb80-a3ed-11e9-90eb-7c64d33f2d5f.png)

@anasereijo what do you think? It's a removal of other nav items.

**Comment by anasereijo** [comment]
Hi @caldav, could you please explain the reason why we are removing the links from the nav? Design wise looks good, just wondering what we are testing. Thank you

**Comment by caldav** [comment]
@anasereijo the goal is to increase the conversion rate of engage forms. Removing the nav is [a pretty common practice](https://duckduckgo.com/?q=landing+page+examples&t=canonical&iax=images&ia=images) for landing pages - that we never tried so far: it's supposed to highlight the content and reduce mental load on the page, leaving the form or the video as the main possible action.

**Comment by anasereijo** [comment]
@caldav makes sense! Should we remove the global nav links too or you think it's necessary? otherwise looks good to me!

**Comment by caldav** [comment]
Maybe as a step 2! Let's start with this one.

**Comment by cristinadresch** [comment]
Results split by devices show that on desktop there is a 5% increase, and on mobile is down by 9%. Interestingly, if you look at the page metrics screenshot you will see the increase in bounce and exit rate is coming primarily from desktop so the drop in form submission isn't related to bounce (it actually reduced slightly on mobile).

@caldav we can discuss this one further tomorrow, it would be good to compare next page visits between control/variation as that could be causing the issue

<img width="1139" alt="Screenshot 2019-08-20 at 14 30 11" src="https://user-images.githubusercontent.com/47972694/63351517-4fce4500-c357-11e9-9335-9f056a6cd8ad.png">
<img width="1142" alt="Screenshot 2019-08-20 at 14 31 12" src="https://user-images.githubusercontent.com/47972694/63351518-4fce4500-c357-11e9-9309-49fab4b2855d.png">
<img width="1143" alt="Screenshot 2019-08-20 at 14 45 31" src="https://user-images.githubusercontent.com/47972694/63352590-53fb6200-c359-11e9-98bb-a508a323712e.png">
<img width="1141" alt="Screenshot 2019-08-20 at 14 45 48" src="https://user-images.githubusercontent.com/47972694/63352591-53fb6200-c359-11e9-9cc8-c095d5c1f35d.png">

**Comment by cristinadresch** [comment]
@caldav to pause and relaunch only with whitepaper as we cant track conversions for webinars

**Comment by caldav** [comment]
Done

**Comment by cristinadresch** [comment]
@cristinadresch to start draft of results for this

**Comment by cristinadresch** [comment]
@caldav to push winning version to 100%

**Comment by caldav** [comment]
@anasereijo to design v2 with hero extending to the top

eg.
![Screenshot from 2019-10-16 16-07-51](https://user-images.githubusercontent.com/18480003/66926909-26682780-f02f-11e9-8c35-7aad248e35ea.png)

**Comment by anasereijo** [comment]
@caldav this is now ready https://github.com/canonical-web-and-design/web-squad/issues/1869 👍

**Comment by cristinadresch** [comment]
Results slides https://docs.google.com/presentation/d/1DRMF3fr3orMtaTj89eYHCCx4qOWZCeY0EmKd1MAOqKU/edit#slide=id.g61d7092969_0_786
